shared/chat/conversation/info-panel/container.js

```diff
@@ -38,8 +38,8 @@ const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
   }
 }
 
-const mapDispatchToProps = (dispatch: Dispatch, {navigateUp}) => ({
-  _onBack: () => dispatch(navigateUp()),
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  _onBack: () => dispatch(Route.navigateUp()),
```

Comment:
why not get it from the 2nd argument and do `navigateUp && navigateUp()` like below?

Reply:
because its not an ownprop,, its not a routable component so its really not there and i didn't want to plumb it through

Reply:
Ah, I see. I think you can just remove this...`onBack` *is* being plumbed through (see `mergeProps` below)

Reply:
Confusingly I think it's  being used and it's using the other navigate up

Reply:
Hmm where is it being used? I don't see `_onBack` referred to from anywhere...

Reply:
I just tested on the simulator and the `onBack` from `mapDispatchToSelectorProps` is the one being called...